src/hooks/useActiveProject.tsx

```diff
@@ -46,6 +50,9 @@ const findProjectAndSubjectById = (
     }
     return getDefault();
   }
+  selectedSubject?.subjectId &&
+    _sdkTracker.identifyUser(selectedSubject?.subjectId);
+  _sdkTracker.userPropertyUpdate('project', selectedProject?.id);
```

Comment:
Feels like this logic should live where the value is actually changed. If `findProjectAndSubjectById` was ever used just to look up the value this would cause a mutation.

Reply:
+1. I also noticed these are the exact same lines as above. It seems like they should live in different places since the subject and project don't necessarily change together.